### PR TITLE
Fix for version comparison from strings to semver

### PR DESCRIPTION
**Pull request by mattfarina**
See #3002 for more detail

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.27.0
+version: 1.27.1
```
Comment (krancour) [inline stable/traefik/Chart.yaml]: Needs to be 1.29.1

Comment (javefang) [reply stable/traefik/Chart.yaml]: Or should it be 1.29.2 now?

**Comment by javefang** [comment]
Hi @mattfarina, this fix is essential for toleration to work correctly on kube 1.10.x. I have tested your change locally and it works nicely. Would be great if we can get it merged!

**Pull request merged**